proto/tendermint/abci/types.proto

```diff
@@ -295,6 +377,21 @@ message TxResult {
   ResponseDeliverTx result = 4 [(gogoproto.nullable) = false];
 }
 
+message TransactionRecord {
+  TxAction action           = 1;
+  bytes tx                  = 2;
+  repeated bytes new_hashes = 3;
```

Comment:
Perhaps not here but somewhere in the spec we need to be explicit that the hashing algorithm matches the one that tendermint is using else this doesn't work

Reply:
I'll add a bullet with this info in the spec.